lib/matplotlib/tests/test_figure.py

```diff
@@ -709,29 +709,30 @@ def test_removed_axis():
     fig.canvas.draw()
 
 
-def test_figure_clear():
+@pytest.mark.parametrize('clear_method', ['clear', 'clf'])
```

Comment:
Souldn't it be enough to test:
```
def test_clf_is_clear():
    fig = plt.figure()
    assert fig.clf is fig.clear
```

Reply:
If we go that route we can do

```
    assert Figure.clf is Figure.clear
```

but I was not sure if we want to pin the test to being a literal alias?  Probably we do....